tests/DialogueLineTests.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+Author: Mr.Sajan
+Date:   10/19/2019
+*/
+
+#include "DialogueLineTests.h"
+#include "TestResult.h"
+#include "../source/DialogueLine.h"
+
+DialogueLineTests::DialogueLineTests(const std::string& n): Testable(n) {}
+
+void DialogueLineTests::build()
+{
+    tests.push([](void) -> TestResult {
+        DialogueLine D1("Hello! ", Person());
+        std::string expected = "Hello! ";
+        std::string result = D1.getLineToSay();
+        bool passed = ( expected == result );
```

Comment:
To have this match the style of everything else, remove the additional spaces, result in `bool passed = (expected == result);`